llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/TileAndDecomposeAttention.cpp

```diff
@@ -283,11 +289,17 @@ createAttentionBody(Value keySlice, Value valueSlice, Value querySlice,
       scaleAccumulator(outputSlice, scaleFactor, loc, builder, ops);
 
   // Compute matmul(softmax, v)
-  auto matmulOp = builder.create<linalg::MatmulOp>(
-      loc, scaledAcc.getType(), ValueRange{partialSoftmax, valueSlice},
-      scaledAcc);
+  Operation *matmulOp;
+  if (transposeV)
+    matmulOp = builder.create<linalg::MatmulTransposeBOp>(
+        loc, scaledAcc.getType(), ValueRange{partialSoftmax, valueSlice},
+        scaledAcc);
+  else
```

Comment:
style-nit: IREE puts braces around all conditionals like this

(correct me if we're using llvm style in llvm-external-projects though)

Reply:
I thought we were following llvm style throughout the codebase, so interesting to know that it is not followed everywhere. I can add braces here though, might be complex enough to deserve it.

Reply:
I only noticed Mahesh making this comment a few times so I'm propagating it. I don't know about llvm-external projects though because it's supposed to be closer to upstream AFAIU, so probably up to you unless someone else can clarify.

Reply:
thanks for clarifying. I will add the braces for now.